string id
string type
string time
string received_time
string received_emergency_cmd
string result
string[] errors
